### DISCUSSION TITLE
Force cpp with kincmake?

### DISCUSSION THREAD

**Discussion by aleph1** [Kode/Kore#588] [Q&A] [answered]
I just updated my project to the latest version of Kode/Kinc and the sources for Kore seem to be excluded due to these lines in Kinc/kincfile.js:

if (!cpp) {
	project.addExclude('Sources/Kore/**');
}

Even if I comment them out it seems to still exclude these sources when running Kinc/make on the Mac. Is there a way to ensure the Kore sources are included?

**Comment by RobDangerous** [comment, marked as answer]
Have a look at https://github.com/Kode/Kinc/blob/master/updates.md - that's the file that always lists all breaking changes (minus the ones I missed).

**Comment by aleph1** [comment]
You rock!

**Comment by RobDangerous** [comment]
You rule!
